### ISSUE TITLE
ERROR: 16: tx-overwinter-expired

### ISSUE THREAD

**Issue by soatok** [radix42/winzec#6]
This isn't really a bug so much as a UI/UX wart that appears to be a bug at first glance.

I was trying to send ZEC from a z-addr to a new t-addr in order to exchange it for BTC transparently (i.e. via Changelly which only supports t-addresses). It kept erroring out with **ERROR: 16: tx-overwinter-expired**.

After waiting for the wallet to sync up to 100%, it succeeded without hiccups. (It was at like 98% when I opened it.)

If this is a reproducible issue, there are a couple of ways to fix it:

1. Block the Send button until the network is at 100%.
2. Queue the transaction silently until the network is at 100%, with an optional "do it now" checkbox to bypass queueing.
3. Telling users to just be patient.

**Comment by radix42** [comment]
yeah you MUST be synced 100% to send from a zaddr due to how they work, which will still be the case with Sapling, so yes, option 1) above is the Right Thing to do....so far everyone has punted to option 3) in your list (on linux and mac too)
